reviews/templates/mail/concept_other_applicants.html

```diff
@@ -0,0 +1,23 @@
+{% extends "mail/base.html" %}
+
+{% block content_nl %}
+Beste collega,<br />
+<br />
+Uw collega {{ creator }} heeft een concept-aanmelding ingediend bij de ethische commissie voor een onderzoek, waar u aan meewerkt.<br />
+Uw concept-aanmelding wordt door de eindverantwoordelijk onderzoeker {{ supervisor }} van deze aanvraag gecontroleerd.
+Zodra deze de concept-aanmelding, evt. na kleine aanpassingen, heeft goedgekeurd en ter toetsing bij de FETC-GW heeft ingediend, krijgt uw collega {{ creator }} van ons bericht.<br />
+Indien de concept-aanmelding nog aanzienlijke wijzigingen uwerzijds behoeft zal de eindverantwoordelijk onderzoeker zelf contact met uw collega opnemen.<br />
```

Comment:
Two points, but very subjective: 

1. I would add 'alleen' here: 'Zodra [..], krijgt alleen je collega [..]'. Just to emphasize this will be the only communication from the portal from here-on
2. The second line could be left out I think. It's factually correct, but not really that interesting for the recipient. Making shorter emails brings along the message across better imo.